src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -3323,6 +3323,7 @@ assign(x = ".rs.acCompletionTypes",
    .rs.makeCompletions(
       token = token,
       results = results.sorted,
-      packages = packages.sorted
+      packages = packages.sorted,
+      type = attr(results, "type")
```

Comment:
Yes I agree, completeme is really somewhat restricted to the information command line R provides, which is only the current line.

I agree it would be nice to provide more information when running in the IDE that could be used for custom completers in the future, but for now I think even this limited functionality is ok.